network/dns_packet.ksy

```diff
@@ -144,6 +148,29 @@ types:
         value: (flag & 0b0000_0000_0000_1111) >> 0
       is_opcode_valid:
         value: opcode == 0 or opcode == 1 or opcode == 2
+  service:
+    seq:
+      - id: priority
+        type: u2
+      - id: weight
+        type: u2
+      - id: port
+        type: u2
+      - id: target
+        type: domain_name
+  txt:
+    seq:
+      - id: length
+        type: u1
+      - id: text
+        type: str
+        size: length
+        encoding: UTF-8
```

Comment:
Are we sure that this comes in UTF-8? My guess would be that DNS protocol longs before predates Unicode, so likely it's treated as raw byte array?

Reply:
Good point.
Let me double check.

Reply:
https://tools.ietf.org/html/rfc1464

RFC says ASCII.
I’ll check whether there is a update to this RFC.

Reply:
I split dns into dns and mdns packet.
It is better to have on dns_packet with UTF-8 if it is possible.

Reply:
Let's stick with UTF-8 for now then, and we can always make it more strict or more relaxed later.